changelogs/DP-23508-5.yml

```diff
@@ -0,0 +1,3 @@
+Changed:
```

Comment:
I havent seen a filename with a dash after the issue number. Seems like its supported so OK.

Reply:
Yeah, this is actually the fifth time I've done that for this ticket so I think we're good. Though it really shows I should have just created new tickets and converted DP-23508 into an epic!